tests/test_hamiltonian.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the expval method of the :mod:`pennylane_lightning_kokkos.LightningKOKKOS` device.
+"""
+import math
+
```

Comment:
As an aside, it may also be good to add tests to the adjoint pipeline showing that this breaks when attempting to use adjoint with expval H currently.

Reply:
Good idea, added.